SevenSwitch.swift

```diff
@@ -201,12 +221,18 @@ import QuartzCore
     private var isAnimating: Bool = false
     private var userDidSpecifyOnThumbTintColor: Bool = false
     private var switchValue: Bool = false
+    lazy var knobSize: CGSize = {
+        let height = self.frame.height - 2
```

Comment:
2 is difference between default knob diameter and height of the UISwitch. Just used library's calculations, but wrapped it to mutable property.